MapboxCoreNavigation/RouteController.swift

```diff
@@ -843,6 +881,15 @@ extension RouteController {
 
         return event.id.uuidString
     }
+    
+    func enqueueFoundFasterRouteEvent() -> String {
+        let eventDictionary = events.navigationFoundFasterRoute(routeController: self)
+        let event = FeedbackEvent(timestamp: Date(), eventDictionary: eventDictionary)
+        
+        outstandingFeedbackEvents.append(event)
```

Comment:
Rather than enqueuing this, I think we can just send this immediately after updating with the new route information (`step`, `newGeometry`, etc), like an arrive event. This event doesn't need to collect locations so doesn't need to be queued and we don't need to set or return a feedback id.